Makefile

```diff
@@ -5,7 +5,17 @@ NAME=doppler
 BINARY=terraform-provider-${NAME}
 # Only used for local development
 VERSION=0.0.1
-OS_ARCH=darwin_$$(uname -m)
+ARCH := $(shell uname -m)
+# required because Terraform always evaluates x86_64
+# architectures as amd64
+ifeq ($(ARCH), x86_64)
+	OS_ARCH=darwin_amd64
+else ifeq ($(ARCH), arm64)
+	OS_ARCH=darwin_arm64
+else
+# can't indent here due to makefile weirdness
```

Comment:
Man, makefiles are silly. It looks like one recommended approach is to use spaces instead of tabs for non-recipe indenting:

```diff
diff --git a/Makefile b/Makefile
index cee460a..15c01d0 100644
--- a/Makefile
+++ b/Makefile
@@ -9,12 +9,11 @@ ARCH := $(shell uname -m)
 # required because Terraform always evaluates x86_64
 # architectures as amd64
 ifeq ($(ARCH), x86_64)
-	OS_ARCH=darwin_amd64
+  OS_ARCH=darwin_amd64
 else ifeq ($(ARCH), arm64)
-	OS_ARCH=darwin_arm64
+  OS_ARCH=darwin_arm64
 else
-# can't indent here due to makefile weirdness
-$(error $(ARCH) is currently untested. Please update the Makefile to handle your architecture properly.)
+  $(error $(ARCH) is currently untested. Please update the Makefile to handle your architecture properly.)
 endif
 
 default: install

```

https://stackoverflow.com/questions/28712585/when-to-use-space-or-tab-in-makefile